lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,533 @@
+const cryptoRandomString = require('crypto-random-string');
+const omit = require('lodash/omit');
+const test = require('ava');
+
+const Collection = require('@cumulus/api/models/collections');
+const Execution = require('@cumulus/api/models/executions');
+const Granule = require('@cumulus/api/models/granules');
+const s3Utils = require('@cumulus/aws-client/S3');
+
+const { secretsManager, dynamodbDocClient } = require('@cumulus/aws-client/services');
+const {
+  CollectionPgModel,
+  destroyLocalTestDb,
+  ExecutionPgModel,
+  fakeCollectionRecordFactory,
+  fakeExecutionRecordFactory,
+  FilePgModel,
+  generateLocalTestDb,
+  GranulePgModel,
+  tableNames,
+} = require('@cumulus/db');
+const { RecordAlreadyMigrated } = require('@cumulus/errors');
+
+// eslint-disable-next-line node/no-unpublished-require
+const { migrationDir } = require('../../db-migration');
+const { migrateGranuleRecord, migrateFileRecord, migrateGranulesAndFiles } = require('../dist/lambda/granulesAndFiles');
+
+const buildCollectionId = (name, version) => `${name}___${version}`;
+
+const dateString = new Date().toString();
+const bucket = cryptoRandomString({ length: 10 });
+
+const generateTestGranule = (collection, executionId, alternateBucket, pdrName, provider) => ({
```

Comment:
I intentionally didn't use `fakeGranuleFactoryV2` because I want to associate the granule record to a collection, whereas `fakeGranuleFactoryV2` calls `fakeGranuleFactory` and sets the collection id to :
```'fakeCollection___v1'```.

Does that make sense or am I missing something in my reasoning? 🤔

Reply:
It does do that, but `fakeGranuleFactoryV2` also allows you to specify an `options` object that lets you override any of the defaults generated by `fakeGranuleFactory`, so it would be pretty easy to override just the `collectionId` to whatever value that you want